engine/swarm/admin_guide.md

```diff
@@ -251,8 +237,87 @@ Before you forcefully remove a manager node, you must first demote it to the
 worker role. Make sure that you always have an odd number of manager nodes if
 you demote or remove a manager
 
+## Back up the swarm
+
+Docker manager nodes store the swarm state and manager logs in the
+`/var/lib/docker/swarm/` directory. In 1.13 and higher, this data includes the
+keys used to encrypt the Raft logs. Without these keys, you will not be able
+to restore the swarm.
+
+You can back up the swarm using any manager. Use the following procedure.
+
+1.  If the swarm has auto-lock enabled, you will need the unlock key in order
+    to restore the swarm from backup. Retrieve the unlock key if necessary and
+    store it in a safe location. If you are unsure, read
+    [Lock your swarm to protect its encryption key](/engine/swarm/swarm_manager_locking.md).
+
+2.  Stop Docker on the manager before backing up the data, so that no data is
+    being changed during the backup. It is possible to take a backup while the
+    manager is running (a "hot" backup), but this is not recommended and your
+    results will be less predictable when restoring. While the manager is down,
+    other nodes will continue generating swarm data that will not be part of
+    this backup.
+
+    > **Note**: Be sure to maintain the quorum of swarm managers. During the
+    > time that a manager is shut down, your swarm is more vulnerable to
+    > losing the quorum if further nodes are lost.The number of managers you
+    > run is a trade-off. If you regularly take down managers to do backups,
+    > consider running a 5-manager swarm, so that you can lose two managers
```

Comment:
nitpick about wording:  I'm not sure whether this is counting the downed backing-up manager when it says that "you can lose two managers while the backup is running".    It implies to me that in addition to the manager that is down (and being backed up), you can lose an additional two.  

But you only lose one more manager while the backup is running, because if your backup is running, you're down to 4/5, and then you can only use 1 more and still maintain quorum.

(also in L263, should there be a space after the period?  `lost.The number`)